utils/ocm.py

```diff
@@ -77,6 +77,47 @@ def _get_cluster_ocm_spec(cluster):
         }
         return ocm_spec
 
+    def create_cluster(self, name, cluster):
+        """
+        Creates a cluster.
+
+        :param name: name of the cluster
+        :param cluster: a dictionary representing a cluster desired state
+
+        :type name: string
+        :type cluster: dict
+        """
+        api = f'/api/clusters_mgmt/v1/clusters'
+        cluster_spec = cluster['spec']
+        cluster_network = cluster['network']
+        ocm_spec = {
```

Comment:
We need to be able to select if a cluster is public/private here. Looking at https://api.openshift.com/#/default/post_api_clusters_mgmt_v1_clusters for the spec

Reply:
done!